cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -219,6 +219,44 @@ if(builtin_xxhash)
   add_subdirectory(builtins/xxhash)
 endif()
 
+#---Check for ZSTD-------------------------------------------------------------------
+# Potentially, this if() block will enable builtin_zstd.
+if (NOT builtin_zstd)
+  message(STATUS "Looking for ZSTD")
+  find_package(ZSTD)
+  if (ZSTD_FOUND)
+    message(STATUS "System ZSTD found; using that.")
+  else()
+    message(STATUS "System ZSTD not found. Switching on builtin_zstd option")
+    set(builtin_zstd ON CACHE BOOL "" FORCE)
+  endif()
+endif()
+
+if (builtin_zstd)
+  set(zstd_version v1.4.0)
+  message(STATUS "Building ZSTD version ${zstd_version} included in ROOT itself")
+  set(ZSTD_URL https://github.com/facebook/zstd/archive/v1.4.0.tar.gz)
```

Comment:
@axel, we have a branch with ZSTD as a ROOT builtins, but after suggestions of Brian we are going simply to add ZSTD  though ExternalProject() to reduce amount of files to be reviewed. After ZTS is landed, we will open next PR with ZSTD as a ROOT builtins.

Reply:
> After ZTS is landed, we will open next PR with ZSTD as a ROOT builtins.

I think that it would be better to replace commits for ExternalProject with the builtin as a last step (before merging the PR), once the code in ROOT has been reviewed. Then we merge only the final version directly.